Libraries/Core/setUpGlobals.js

```diff
@@ -21,6 +21,10 @@ if (global.window === undefined) {
   global.window = global;
 }
 
+if (global.self === undefined) {
```

Comment:
whatwg-fetch@3 relies on the `self` global

Reply:
Maybe worth putting this comment in the code?

Reply:
I didn’t since it’s not specific to the fetch polyfill and is just a standard dom api like Window (https://developer.mozilla.org/en-US/docs/Web/API/Window/self). I don’t mind adding it though if you prefer.